core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -56,17 +59,64 @@ public void testHighway() {
         edge = GHUtility.createMockedEdgeIteratorState(0, flags);
         assertEquals("_default", encoder.getHighwayAsString(edge));
     }
+    
+    @Test
+    public void testTunnel() {
```

Comment:
Strange that this appears as change, as it is already there: https://github.com/graphhopper/graphhopper/blob/master/core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java#L61-L103